streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/InternalTopicIntegrationTest.java

```diff
@@ -73,32 +77,39 @@
 @Timeout(600)
 @Tag("integration")
 public class InternalTopicIntegrationTest {
-    public static final EmbeddedKafkaCluster CLUSTER = new EmbeddedKafkaCluster(1);
+
+    public static EmbeddedKafkaCluster cluster = null;
 
     @BeforeAll
     public static void startCluster() throws IOException, InterruptedException {
-        CLUSTER.start();
-        CLUSTER.createTopics(DEFAULT_INPUT_TOPIC, DEFAULT_INPUT_TABLE_TOPIC);
+
+        final Properties props = new Properties();
+        props.setProperty(GroupCoordinatorConfig.GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG, "classic,consumer,streams");
+        props.setProperty(ServerConfigs.UNSTABLE_API_VERSIONS_ENABLE_CONFIG, "true");
+
+        cluster = new EmbeddedKafkaCluster(1, props);
```

Comment:
What is the difference between creating the embedded cluster here or on line 81?

Reply:
We need to pass those props into it. I guess we could do it with a `static {` block, but I like this more. We should also not make it final to do a `cluster = null` in @AfterAll, otherwise the cluster will stay in memory for the rest of the test suite, IIUC. Is there any magic in JUnit that unloads test classes after they have been run? As far as I know, there is not. So if we have to use static variables in test classes, we should clear them out in @AfterAll.

Reply:
Good points!